float32 y_min
float32 y_max
---